roles/libvirt_manager/tasks/ocp_layout.yml

```diff
@@ -0,0 +1,66 @@
+---
+## Use a dedicated port, this instance will be removed
+# once OCP cluster is deployed.
+- name: Ensure VBMC is present
+  vars:
+    cifmw_virtualbmc_daemon_port: 51881
+  ansible.builtin.import_role:
+    name: "virtualbmc"
+
+- name: Manage virtual resources
+  vars:
+    cifmw_libvirt_manager_storage_pool: "oooq_pool"
+    action: "create"
+    cifmw_libvirt_manager_pool_dir: "/home/dev-scripts/pool"
+  block:
+    - name: Create storage pool directory
+      become: true
+      ansible.builtin.file:
+        path: "{{ cifmw_libvirt_manager_pool_dir }}"
+        state: directory
+        owner: "{{ ansible_user_id }}"
+
+    - name: Create pool in libvirt
+      ansible.builtin.import_tasks: "storage_pool.yml"
+
+    - name: Craft layout and pass it to create_vms.yml
+      vars:
+        cifmw_libvirt_manager_vm_template: "ocp_domain.xml.j2"
+        cifmw_libvirt_manager_vm_prefix: ''
+        cifmw_libvirt_manager_vm_separator: '_'
+        vm_type: 'ocp_master'
+        _ocp_layout:
+          vms:
+            ocp_master:
```

Comment:
I think needs to be a copy of `cifmw_libvirt_manager_layout.vms.ocp` then remove the xml_paths and re-write the `disk_file_name` to `blank`.
See: OSPRH-7489

Reply:
hmm, lemme see if that'd be easy.

Reply:
could get it right. Note that the current (and coming) way to manage those extra volumes for OCP nodes isn't good, since they aren't recreated when you reuse an env.... Now that we get out of the devscripts world, a follow-up can simplify and make that thing better in many ways (with overlays for OCP extra volumes, for instance).